lib/rootTree/convertRootTreeProofToBuffer.js

```diff
@@ -0,0 +1,30 @@
+const BufferWriter = require('@dashevo/dashcore-lib/lib/encoding/bufferwriter');
+const { convertBitArrayToUInt8Array } = require('@dashevo/dashcore-lib/lib/util/bitarray');
+
+function convertRootTreeProofToBuffer(rootTreeProof) {
+  // amount of subtrees to proof
+  const totalObjects = 1;
```

Comment:
I would add a comment explaining why this is exactly equal to 1 and why keep it. It is equal to 1 because at the moment our proof won't be a simultaneous proof for more than 1 tree, i.e. it's always identities OR documents OR contracts, not an AND in any case, so there's always only 1 leaf to prove. The libraries that perform verification need to know this, as this is equal to 1 in this particular case, usually it's not equal to 1.